tests/test_v2.py

```diff
@@ -166,6 +162,18 @@ def test_v2_filters_codecs(filters: Any, order: Literal["C", "F"]) -> None:
     np.testing.assert_array_equal(result, array_fixture)
 
 
+@pytest.mark.filterwarnings("ignore")
+def test_create_array_defaults():
+    store = zarr.storage.MemoryStore()
```

Comment:
```suggestion
@pytest.mark.filterwarnings("ignore")
@pytest.mark.parametrize('store', ['memory'], indirect=True)
def test_create_array_defaults(store: Store):
"""
Test that passing compressor=None results in no compressor. Also test that the default value of the compressor
parameter does not produce no compressor.
"""
```
I like test docstrings, and the fixture makes it easier to make it easier to enact sweeping changes to all the tests that use stores.